src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          autocomplete="off"
+          >
+        </div>
```

Comment:
1) use label instead of div
2) apply code styling [recommendations](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
        <label class="form-field">
          Surname:
          <input
            type="text"
            name="Surname"
            autocomplete="off"
          >
        </label>
```

Reply:
Hi, i need chenge oll div to label ?